operator/apis/lifecycle/v1alpha3/common/common_test.go

```diff
@@ -292,8 +292,8 @@ func Test_GenerateTaskName(t *testing.T) {
 		},
 		{
 			Check: PreDeploymentCheckType,
-			Name:  "loooooooooooooooooooooooooooooooooooooong_name",
-			Want:  "pre-looooooooooooooooooooooooooooooo-",
+			Name:  "loooooooooooooooooooooo00000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000ooooooooooooooooooooong_name",
```

Comment:
can we move this loooooooooooong guy above in a const?

Reply:
sure